proxysql/build-binary-proxysql

```diff
@@ -9,6 +9,30 @@
 set -o errexit
 set -o xtrace
 
+if [ -f /usr/bin/yum ]; then
+    RHEL=$(rpm --eval %rhel)
+    ARCH=$(echo $(uname -m) | sed -e 's:i686:i386:g')
+    if [[ $RHEL -eq 8 ]]; then
+        sudo yum -y install python2 gnutls-devel libtool || true
+    fi
+    sudo yum -y install wget
+    #
+    RHEL=$(rpm --eval %rhel)
+    #
+    sudo wget -O /etc/yum.repos.d/percona-dev.repo http://jenkins.percona.com/yum-repo/percona-dev.repo
+    if [[ $RHEL -eq 6 ]]; then
+        sudo yum -y install percona-devtoolset-gcc percona-devtoolset-binutils percona-devtoolset-gcc-c++
+        sudo yum -y install automake bzip2 cmake make g++ gcc git openssl openssl-devel gnutls libtool patch
+        sudo yum -y install gnutls-devel libtool || true
+    	  sudo yum -y install epel-release
+    	  sudo sed -i "s/mirrorlist=https/mirrorlist=http/" /etc/yum.repos.d/epel.repo
+    	  sudo yum -y install http://repo.okay.com.mx/centos/6/x86_64/release/okay-release-1-1.noarch.rpm || true
+    	  sudo yum -y upgrade automake autoconf
+    else
+        sudo yum -y install automake bzip2 cmake make gcc-c++ gcc git openssl openssl-devel gnutls gnutls-devel libtool patch
```

Comment:
I think it could be moved to percona-lab/ps-build as it uses this docker image

Reply:
ProxySQL uses perconalab/pxc-build docker image and all these packages are present in PXC install-deps file, but proxysql build is failing on CentOS 8. So I have added these packages explicitly in build script.